zaas-client/src/main/java/org/zowe/apiml/zaasclient/exception/ZaasConfigurationErrorCodes.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright Contributors to the Zowe Project.
+ */
+package org.zowe.apiml.zaasclient.exception;
+
+public enum ZaasConfigurationErrorCodes {
+    TRUST_STORE_NOT_PROVIDED("ZWEACZ500E", ""),
```

Comment:
Aren't we using `ZWEAC***` for Api Catalog messages? I think Security error codes related should have different message code. Also we may want to use the same length to be consistent with other messages, so 7 chars.. What do you think about it?

Reply:
Honestly I don't know, I was going with the messages based on the ZaasClientErrorCodes
Do we have anywhere some documentation with respect to what codes we already used and what we can use?